### ISSUE TITLE
export audio

### ISSUE THREAD

**Issue by trosel** [maxkrieger/voiceliner#55]
i'd like to be able to export the whole audio outline as one big audio file. right now you can only export the text version.

i could see this as a really neat way to put an entire podcast or audio essay together, for example.

maybe for phase 1, the app gives you a download of all the individual audio clips (preserving the order they are in in the outline). 

phase 2, maybe the app gives the option to combine the audio for you into one big file. but it's not strictly needed because there are other services that can do this like this one https://clideo.com/merge-audio

**Comment by benjaffe** [comment]
I have a version of this working locally, but I haven't found time to figure out how to test on my real iPhone. That said, @maxkrieger , if you still want a PR, I'm happy to submit it. You'd probably need to give it a test drive for me though.

**Comment by benjaffe** [comment]
Actually, I'd love your thoughts on this @maxkrieger . Here's what I have:
This: 
<img width="334" alt="image" src="https://user-images.githubusercontent.com/573204/157566779-d4e98aca-f85b-42ed-a7ad-2c5c81bc85bd.png">

exports this file structure, with md exported normally:
<img width="348" alt="image" src="https://user-images.githubusercontent.com/573204/157566746-fcbb26fe-1a43-488a-8e1a-17d4c0305a89.png">
.....
and this:
<img width="334" alt="image" src="https://user-images.githubusercontent.com/573204/157566846-d8538487-7426-497d-bf00-b760525a26e2.png">

exports the same files, with this md:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/573204/157566895-4424dbd9-e206-47d2-a844-7e2215e37e3f.png">

This linking should work if the user brings the `.md` into Logseq, and adds the audio files to `<logseq-dir>/assets`

This is great for me, but I suspect it's a bit too specific for general users of VoiceLiner. Do you have guiding principals around what should and shouldn't be in the app? And what would you prefer I do in this case for associating the text in the `.md` with the audio?

**Comment by benjaffe** [comment]
I can do all sorts of things since I have a server with a file watcher, but I do need some way to know what audio file is associated with which line. Some ideas:
- Maybe I could link like in the last screenshot, but without the `../assets/`
- Perhaps the file name could be put in parenthesis at the end, so it's not so tool-specific?

As far as testing, it's mostly that I've never done iOS development, so the trivial roadblocks I hit take a bunch more effort because I don't even know what I don't know. :P

**Comment by maxkrieger** [comment]
@benjaffe on second thought the ../assets seems fine as default behavior for the link audio files if you export the audio files in an adjacent ./assets folder maybe? Just don't specify logseq, pretty sure other knowledge tools use this directory hierarchy too :D


Re running on iOS, having Xcode open and plugging in your phone is a good first step, these instructions might help https://medium.com/front-end-weekly/how-to-test-your-flutter-ios-app-on-your-ios-device-75924bfd75a8

**Comment by benjaffe** [comment]
Ah, forgot to reply to this. It's funny, we've flipped on this. I set it to link files in the same file as the .md, since that's what gets exported. 

We could always make a folder structure instead, but I'm now thinking that'd be more trouble than it's worth (since it'd require merging the folders, and I also worry that a user could accidentally drag the exported assets folder over to their logseq folder, nuking the original one). It mirrors Logseq better to have an assets folder, but it doesn't make things easier for most workflows I can imagine.
